streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -260,7 +260,8 @@ public void shouldProcessingFromSinkTopic() {
         }
 
         // no exception was thrown
-        assertEquals(Collections.singletonList("A:aa (ts: 0)"), processorSupplier.theCapturedProcessor().processed);
+        assertEquals(asList(new KeyValueTimestamp<>("A", "aa", 0)),
```

Comment:
why do we need to avoid singleton list? Also could we format the assertion call more consistently like in L285-286

Reply:
done